lib/maglev/listeners/broadcaster.rb

```diff
@@ -186,7 +186,12 @@ def broadcast_listeners
           end
 
           if !listened and event.targets.any?
-            raise EventError.new("event target was specified for #{listener.class.name} that does not support the #{event.name} event")
+            error = "event target was specified for #{listener.class.name} that does not support the #{event.name} event"
+            if MagLev.production?
+              warn error
```

Comment:
Actually, we really want it to be `logger.report(:warn, error)` because report will also create a Sentry ticket.